redfish-core/lib/log_services.hpp

```diff
@@ -4233,6 +4233,71 @@ inline void deleteSystemHardwareIsolationLogEntryById(
         entryObjPath.str, hwIsolationEntryIfaces);
 }
 
+/**
+ * @brief API Used to deisolate the all HardwareIsolation entries.
```

Comment:
What is the context of deisolate  ? POST command or DELETE command ? can you please use different terms here ? to clearly differentiate between Delete and DeleteAll action?

Commit message has no difference in the output for "Get the isolated hardware entry after deisolation."